projects/valtimo/components/src/lib/components/search-fields/search-fields.component.html

```diff
@@ -142,7 +142,7 @@
 >
   <v-date-picker
     *ngIf="searchField.dataType === 'date' && searchField.fieldType === 'single'"
-    [title]="'searchFields.' + searchField.key | translate"
+    [title]="getTitle(searchField)"
```

Comment:
Using a function like this in the template often leads to problems. (`getTitle`). Move the logic inside the template instead by declaring `let` etc or using ngIf. Now you have an infinite console.log because the functions keep running:

![image](https://user-images.githubusercontent.com/79840403/201871386-4ae0fe25-4461-4e91-8101-ad8acdad5a79.png)